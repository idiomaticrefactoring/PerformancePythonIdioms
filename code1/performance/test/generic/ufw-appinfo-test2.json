[{"profile":"TEST2","title":"My test app2","description":"a longer description of the test app here.","tcp_ranges":[{"start":0,"end":65535}],"udp_ranges":[{"start":50,"end":51}],"tcp_list":[53],"udp_list":[53],"normalized_tcp_ranges":[{"start":0,"end":65535}],"normalized_udp_list":[53],"normalized_udp_ranges":[{"start":50,"end":51}]}]
