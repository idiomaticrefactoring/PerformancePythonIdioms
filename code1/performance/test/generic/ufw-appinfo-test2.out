Profile: TEST2
Title: My test app2
Description: a longer description of the test app here.

Ports:
  any/tcp
  50:51/udp
  53

